WebAPI/WebComponents/Meeting/src/SyncBar.js

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react';
+
+export default function SyncBar(props) {
+  const [inputValue, setInputValue] = useState('');
+  const { meetingId } = props
+
+  const submitSync = (videoposition) => {
+    const request = {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+        'Authorization': 'Bearer ' + localStorage.getItem("userToken")
+      },
+      body: JSON.stringify({
```

Comment:
Timestamp is not set here?

Reply:
timestamp added as ISOString

Reply:
It should be start time of the agenda point 2. We do not get the timestamp from the backend yet. So I think we should postpone merging this PR until we have the correct timestmap